.local/render-matrix3.py

```
@@ -0,0 +1,166 @@
+Each run dir under $MATRIX has samples.csv. Produces:
+  - $MATRIX/<run>.png — multi-panel time-series for that run
+  - $MATRIX/summary.png — bar charts of headline metrics across runs
+  - $MATRIX/summary.txt — text table
+"""
+import sys, os, csv, glob, re
+from pathlib import Path
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+MATRIX = sys.argv[1] if len(sys.argv) > 1 else sorted(glob.glob('/tmp/matrix3-2*'))[-1]
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/matrix3-2*"

(S108)